packages/core/test/foundry/optimistic-assertor/examples/DataAsserter.t.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+import "../Common.sol";
+import "../../../../contracts/optimistic-assertor/implementation/examples/DataAsserter.sol";
+
+contract DataAsserterTest is Common {
+    DataAsserter public dataAsserter;
+    bytes32 dataId = bytes32("dataId");
+    uint256 correctData = 1000;
+    uint256 incorrectData = 2000;
+
+    function setUp() public {
+        _commonSetup();
+        dataAsserter = new DataAsserter(address(defaultCurrency), address(optimisticAssertor));
+    }
+
+    function test_DataAssertionNoDispute() public {
+        // Assert data.
+        vm.startPrank(TestAddress.account1);
+        defaultCurrency.allocateTo(TestAddress.account1, optimisticAssertor.getMinimumBond(address(defaultCurrency)));
+        defaultCurrency.approve(address(dataAsserter), optimisticAssertor.getMinimumBond(address(defaultCurrency)));
```

Comment:
Might consider moving this to `setup` as its shared by all tests

Reply:
Maybe we can do a different PR to group this kind of things in all the tests?